cms/static/cms/js/modules/cms.structureboard.js

```diff
@@ -112,6 +112,12 @@ $(document).ready(function () {
 				}
 			});
 
+			//fix issue #4407
+			$('.cms-toolbar-expanded .cms_dragarea .cms_draggables .cms_draggable').bind('cms.structureboard.menustate.change', function(e){
+				var drag    = $(this);
```

Comment:
I am aware that the .on() method is the preferred method for attaching event handlers, 
nonetheless .bind() is systematically used in cms.plugins.js
I changed it as requested, but it is not consistent